### PR TITLE
Address #1118

### PR DESCRIPTION
**Pull request by DavidLitwin**
Small change to gw component flux calculations, correctly indexing area_of_cell.

### CODE REVIEW

**Comment by kbarnhart** [comment]
@DavidLitwin I think part of the problem is that area_of_cell is a number of cells long array, so indexing is tricky if there are inactive cells. I switched to using area_of_cell_at_node, so it can be indexed correctly be self._cores. 

Could you add a test of this part of the code that has a grid with at least one interior node which is inactive (so that grid.node_at_cell is not the same as grid.core_nodes)

**Comment by DavidLitwin** [comment]
@kbarnhart Ah, I didn't know that method existed. Looks good though! I'll see what I can do as far as a test.

**Comment by kbarnhart** [comment]
[Linking to the JOSS Review Issue](https://github.com/openjournals/joss-reviews/issues/1935)

**Comment by DavidLitwin** [comment]
@kbarnhart I'm getting an error with this method.

`AttributeError: 'RasterModelGrid' object has no attribute 'area_of_cell_at_node'`

Am I missing something?

**Comment by kbarnhart** [comment]
you are right... I mistyped. it is cell_area_at_node

File: tests/components/groundwater/test_dupuit_percolator.py [hunk 1]
```diff
@@ -219,3 +219,31 @@ def test_wt_above_surface_adaptive_run_step():
     gdp = GroundwaterDupuitPercolator(grid, recharge_rate=0.0)
     gdp.run_with_adaptive_time_step_solver(1)
     assert_equal(wt[4], 1)
+
+
+def test_inactive_interior_node():
```
Comment (kbarnhart) [inline tests/components/groundwater/test_dupuit_percolator.py]: looks good. thanks @DavidLitwin

**Comment by kbarnhart** [comment]
Addressed #1118

**Pull request merged**